README.md

```diff
@@ -179,6 +205,29 @@ $ gto show churn
 ╘════════════╧═══════════╧═════════╧═════════════════════╧═══════════════════╧══════════════╛
 ```
 
+#### Enabling Stages/Kanban workflow
+
+In some cases, you would like to have a latest label for an artifact version to
+replace all the previous labels. In this case the version will have a single
+label. This resembles Kanban workflow, when you "move" your artifact version
+from one column ("label1") to another ("label2"). This is how MLFlow and some
+other Model Registries works.
+
+To achieve this, you can use `--last-label-for-version` flag (or `--last` for
```

Comment:
Just my take since we discussed this: I don't think the long `--last-label-for-version` flag is needed, and the help description for the flag is probably enough to specify that `--last` means the last label for the version (not the last version for the label).

Also, we use `latest` in other places, so it might be worth considering whether `last` and `latest` have different meanings (in which case we might want to clarify each), or else use one of those words consistently.

Reply:
Yes, they have different meanings. I had an idea of introducing two flags: `gto show --greatest` (sort by semver) and `gto show latest` (sort by timestamp) with `greatest` as a default.

Re options: Maybe `--last-stage` or `--latest-stage` could work?

Reply:
What about `--hide-old-stages`? Anyway, to be clear, it shouldn't block the PR.

Reply:
@aguschin @dberenbaum 
Suggestion - How about we use a per-label prefix that will make this specific label behave like a stage?
We would be able to support behaviors in parallel:
- free-form labels (user needs to assign and remove)
- stages (assigning a stage overrides previous stages assignments per-artifact-version)
- environments (assigning an environment to an artifact-version will cause this environment to be "unassigned" from other versions of that artifact)

Reply:
That summary of the three different behaviors is great @omesser.

TBH I like the current approach by @aguschin where you can switch between those by choosing how to view the labels rather than having it coded into the tag names. It makes it much easier to play around with the different approaches without being locked into one of them. The only part I don't like is that naming/explaining is not that easy, but I don't think it would be easy no matter what approach we take.

Reply:
I think the main downside of having this determined by the "querying"/"viewing" command is that you cannot mix and match. This prevents from using GTO to manage environments if you already use it for kanban-like stages and vice versa.
To put it in another way - if I build my workflow and CI to treat gto labels like kanban-stages, it will never make sense to view those as something else. If I use them as deployment environments - it will never make sense to query them as anything else. querying it the wrong way is only room for error, there is not upside to that flexibility after the label mechanism was decided on assignment.

Reply:
@omesser, I agree with your argument, but I think flexibility is more important currently for us to see what people want. Also, an important question you raised is how this will behave in Studio when a team collaborates. A misunderstanding may appear when different users interpret the labels with different mechanics (e.g. one with free-form labels and one with stages).

On the second point, I don't think I'll be able to implement this before Studio release, and I doubt it will be handy without Studio BE/FE support cause it just won't be shown there. So let's continue to discuss this and gather some users' feedback to see how they want this to work, and then let's implement this in a separate PR.

Reply:
Seems like we don't have much choice in the short term. In the medium/long term, separate concepts of "stages" and "environments" seem useful.